.eslintrc.js

```diff
@@ -5,10 +5,15 @@ module.exports = {
     sourceType: 'module'
   },
   env: { browser: true },
-  extends: ['airbnb-base', 'plugin:vue/essential', 'plugin:cypress/recommended', 'standard'],
+  extends: [
+    'airbnb-base',
+    'plugin:vue/essential',
```

Comment:
strongly-recommended contient essential `Enforce all the rules in this category, as well as all higher priority rules`